src/SBitSet.jl

```diff
@@ -0,0 +1,284 @@
+struct SBitSet{N}
+    chunks::NTuple{N, UInt64}
+end
+
+mutable struct MBitSet{N}
+    chunks::NTuple{N, UInt64}
+end
+
```

Comment:
It would be tempting to use `NTuple{N, VecElement{UInt64}}` here. Perf with plain UInt64 looks ok, though. Diverging opinions?

Reply:
From memory (and I hope I'm remembering this right!), when I started this work, LLVM vectors of odd lengths (beginning at ~7?) wouldn't work... does anyone know whether they are safe/efficient for arbitrary `N` these days?

Reply:
>LLVM vectors of odd lengths (beginning at ~7?) wouldn't work...
In these cases julia is smart enough to emit a working fallback llvm array (like `NTuple{N, UInt64}`). So that is not blocking.

On the other hand,  `NTuple{N, VecElement{UInt64}}` is inconvenient to use, and llvm should optimize the array to vector anyways.

So we could probably do either and be OK.